install/helm/agones/README.md

```diff
@@ -3,3 +3,69 @@
 This chart installs the Agones application and defines deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 See [Install Agones using Helm](https://agones.dev/site/docs/installation/install-agones/helm/) for installation and configuration instructions.
+
+## Development Work on Agones Custom Resource Definitions (CRDs)
```

Comment:
This feels like a really weird place - I feel like this is a little more user-centric README in general.

I'm struggling to come up with a better place, though. @markmandel ?

Reply:
Would it make sense to follow the same pattern as the SDK and link to the ReadMe from the contributing page? https://github.com/googleforgames/agones/blob/46c5ef9f7433004c3fc1721b405793f26d5d224f/README.md?plain=1#L56

Reply:
This has both a style guide as well as a development guide - which is not a bad thing, but we've never quite done it like this, so it's a little trickier to put in an existing place.

> Would it make sense to follow the same pattern as the SDK and link to the ReadMe from the contributing page?

What README would you put it in? There isn't really a spot that I can think of. We could always add a `docs` folder to https://github.com/googleforgames/agones/tree/main/build though? But open to your suggestions.

It could possibly be a section in https://github.com/googleforgames/agones/blob/main/build/README.md#testing-and-building ?

Or take a stab at taking apart https://github.com/googleforgames/agones/blob/main/build/README.md into some sub-sectioned files, even it's just some basics - and it could be the beginning of making that a better experience - right now it's not terrific.